whois-api/src/main/java/net/ripe/db/whois/api/rest/SyncUpdatesHttpSchemeFilter.java

```diff
@@ -0,0 +1,37 @@
+package net.ripe.db.whois.api.rest;
+
+import jakarta.servlet.Filter;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.ServletRequest;
+import jakarta.servlet.ServletResponse;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import org.eclipse.jetty.http.HttpStatus;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+
+@Component
+public class SyncUpdatesHttpSchemeFilter implements Filter {
+
+    @Value("${syncupdates.http.error:false}")
```

Comment:
may be syncupdates.http.allowed?

Reply:
I leave this as it is but changed the variable to
`httpError`